### PR TITLE
Detecting a arbitrary word

### PR DESCRIPTION
**Pull request by dward118**

### CODE REVIEW

File: utils/data.py [hunk 1]
```diff
@@ -12,8 +12,10 @@ def __init__(self, *args, prefix=None, **kwargs):
     async def on_message(self, msg):
         if not self.is_ready() or msg.author.bot or not permissions.can_handle(msg, "send_messages"):
             return
-
-        await self.process_commands(msg)
+        if await listener.check_for_profanity(msg):
+            await msg.channel.send("Bad word detected!")
```
Comment (sethsarchet) [inline utils/data.py:20]: Overall, this does look pretty good. Maybe you could be a bit more specific on what you say when a bad word is detected. Like, you could have it say the name of the user that said the bad word, then say something about that language not being permissible on the server.

Comment (jcgoble3) [reply utils/data.py:20]: Good thoughts, but detailed handling of bad words is out of scope for this task. It would be handled in #40. This task is to just demonstrate the ability to filter for a word (any arbitrary word or words) and do something (anything) in response.

Comment (NateDawg96) [inline utils/data.py:20]: The code looks good. I do think that it might be appropriate in the response to indicate which word was bad. Not only so that we know which word it detected, but maybe also so any user would know which word to not say anymore. Really the code looks great and I'm nitpicking.

File: utils/listener.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+bad_words = ['fuck', 'ass', 'penis', 'shit']
+
+
+async def check_for_profanity(message):
+    return any([bad_word in message.content.lower().split() for bad_word in bad_words])
```
Comment (NateDawg96) [inline utils/listener.py:5]: Will this code detect a bad word regardless of whether it's typed in upper or lower case? Or does it only work for lowercase words that match exactly those in the list?

Comment (NateDawg96) [reply utils/listener.py:5]: You know what I just looked again and realized it does do that. Disregard my comment above.

File: config.json.example [hunk 3]
```diff
@@ -1,15 +0,0 @@
...
-  ],
-  "prefix": [
-    "!"
-  ],
-  "activity": "games!",
-  "activity_type": "playing|watching|listening|competing",
-  "status_type": "online|idle|dnd",
-  "alexflipnote_api": "nice_token_you_got_here",
-  "version": "2.0.2"
-}
```
Comment (jcgoble3) [inline config.json.example]: Why is this file deleted? This is an important file as it shows the developer how to create their own config.json. Please put it back.

**Review by jcgoble3 — CHANGES_REQUESTED** [review]

File: config.json.example (hunk 3 above)
Comment (jcgoble3) [reply config.json.example]: If you renamed the file to create your local config.json, you should have made a copy instead.

Comment (dward118) [reply config.json.example]: I added the config file back into the repository and pushed the changes.

File: utils/data.py (hunk 1 above)
Comment (jcgoble3) [reply utils/data.py:20]: I think you really shouldn't repeat the bad word in an open channel. Especially if it's something really bad like the n-word or other slurs. Logging it in an private admin channel would be reasonable, but out of scope for this task (suitable for #40, though).

**Review by jcgoble3 — APPROVED** [review]
All good to me.
